plugins/net_plugin/net_plugin.cpp

```diff
@@ -935,12 +941,13 @@ namespace eosio {
          break;
       }
       default:
+         fc_dlog(logger, "sending empty request but not calling sync wait on ${p}", ("p",peer_name()));
          enqueue( ( sync_request_message ) {0,0} );
       }
    }
 
    void connection::cancel_fetch() {
```

Comment:
It looks like the only call to `cancel_fetch` was removed in favor of an exact copy of this implementation (see line 1059) . Should that line call this method, or should this method be removed?

Reply:
I removed it. I also removed the redundant logger instances that I commented out in this pr.